roadmap.md

```diff
@@ -1,27 +1,29 @@
 # Roadmap
 
-Below is the current roadmap and timeline for RDSS, summer 2022 through January 2023. It is subject to change as needed, however it is written in markdown, so all history is recorded.
+Below is the current roadmap and timeline for RDSS. It is subject to change as needed, however it is written in markdown, so all history is recorded.
 
 ```mermaid
 gantt
-    title RDSS 2023
+    title RDSS Roadmap
     axisFormat  %Y-%m
     dateformat YYYY-MM
     section PDC
-    MVP software stabilizing: pd1,2023-05-01, 92d
     MVP testing and launch support: pd2,2023-07-31,62d
-    PDC post-MVP features: pd3,2023-10-01,31d
     section TigerData
-    TigerData training and MVP: td1,2023-08-08,54d
-    TigerData MVP software: td2,2023-11-01,61d
+    TigerData MVP software: td1,2023-08-08,85d
```

Comment:
Should this box get cropped to reflect that we are dropping tigerdata MVP in favor of PDC migration and launch support and ORCID work?

Reply:
The way it is right now it stops only for the short segment for the Cross-IT research and development work, but it continues otherwise. That matches what I thought we were going to do.

Reply:
I don't think so; we picked up TigerData work again at the beginning of August (Kraken sprint) and we're also working on the project form feature during the ORCiD work now, so we are still working toward TigerData MVP during this time.  Thanks for checking.